docker/Makefile

```diff
@@ -3,18 +3,77 @@ ROOT_IMAGE ?= alpine:3.12
 CERT_IMAGE := alpine:3.12
 GOLANG_IMAGE := golang:1.15-alpine
 
-BASE_IMAGE := localhost/baseimg:$(VERSION)-$(shell echo $(ROOT_IMAGE) | tr : -)
-DEBUG_IMAGE := localhost/debugimg:$(VERSION)-$(shell echo $(GOLANG_IMAGE) | tr : -)
+BASE_IMAGE := $(DOCKER_NAMESPACE)/baseimg:$(VERSION)-$(shell echo $(ROOT_IMAGE) | tr : -)
```

Comment:
As a good practice, define this var in the beginning of the Makefile, defaulting to the env var:

```Makefile
DOCKER_NAMESPACE?=jaegertracing
```

Same with the other one (`DOCKER_BUILD`).

Reply:
this is a sub Makefile. 
should we need to re-define those twice?